frontend/src/utils/date/index.js

```diff
@@ -4,4 +4,16 @@ export const formatLocalDate = ({ year, month, day }) => {
   return `${year}-${monthFormatted}-${dayFormatted}`
 }
 
+export const parseLocalDateTime = localDateTime => {
+  const year = localDateTime.getFullYear().toString()
+  const month = addZeroPrefixIfLessThanTen((localDateTime.getMonth() + 1).toString())
```

Comment:
```suggestion
  const month = (localDateTime.getMonth() + 1).toString().padStart(2, '0')
```
이런 식으로 자바스크립트가 제공하는 함수를 쓸 수 있을 것 같아요!

Reply:
음 padStart 부분을 명시적으로 표시하도록 
`const addZeroPrefixIfLessThanTen = value => value.padStart(2, "0")` 메서드 분리 한건데요...!
그냥 padStart를 바로 사용하는 것이 더 보기 좋을 수 있겠네요 😄

Reply:
앗! 그 부분을 주의깊게 못 봤네요.
그런 목적이라면 지금대로도 괜찮을 것 같아요!